### DISCUSSION TITLE
How do I make my plotly chart in a blazor webassembly project resize when window width is changed?

### DISCUSSION THREAD

**Discussion by iancoetzer** [LayTec-AG/Plotly.Blazor#426] [Q&A]
Hi
Currently when i resize my browser window everything else, mudblazor content resizes neatly.
however the chart remains off screen?

![image](https://github.com/LayTec-AG/Plotly.Blazor/assets/7976311/de2b7704-06ab-4fd9-ae6e-0a0de66f45bd)

how can i make it responsive so that it adjusts its width same as other components? is there somewhere i can set its width to be 100% of the width of its parent container?

![image](https://github.com/LayTec-AG/Plotly.Blazor/assets/7976311/8d1a0b6b-ccf9-46e2-b1c2-8eca4e7eb5b9)

**Comment by erth-ft** [comment]
It would be really nice to able to have it follow an outer div that has css `resize:both;` or similar.

**Comment by volmart** [comment]
Did you configure responsive option?
`Config _config = new()
{
        Responsive = true,
}` 

It suppose to make plotly control responsive to window resizing
